packages/preview/modern-cornell-notebooks/0.2.0/typst.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "modern-cornell-notebooks"
+version = "0.2.0"
+entrypoint = "template/cornell.typ"
```

Comment:
This should point to `lib/lib.typ` instead (or at least to a file that is not part of your template).

```suggestion
entrypoint = "lib/lib.typ"
```